### DISCUSSION TITLE
Release plan question?

### DISCUSSION THREAD

**Discussion by mestinso** [ThummeTo/FMI.jl#243] [General]
I read the updated readme with the release plans targeting 1.0, however one thing that wasn't clear was whether you will be registering the releases along the way (0.14, 0.15, ..., 1.0)? For example, if FMI.jl is installed now, FMI 0.13.3 installs rather than the new 0.14. Thanks!

**Comment by ThummeTo** [comment]
Now v0.14.0 is registered :-)
Plan (as you say) is to release 0.14, 0.15, 0.16 ... and finally 1.0
The 0.14 is the worst breaking change, I think until 1.0 there are no heavy breaking changes to expect
